Content.Shared/CCVar/CCVars.Vote.cs

```diff
@@ -177,4 +177,26 @@ public static readonly CVarDef<int>
     /// </summary>
     public static readonly CVarDef<bool> VotekickIgnoreGhostReqInLobby =
         CVarDef.Create("votekick.ignore_ghost_req_in_lobby", true, CVar.SERVERONLY);
+
+    /*
+    * AUTOVOTE SYSTEM
+    */
+
+    /// ADT TWEAK START
+
+    /// Enables the automatic voting system.
+    public static readonly CVarDef<bool> AutoVoteEnabled =
+        CVarDef.Create("vote.autovote_enabled", false, CVar.SERVERONLY);
```

Comment:
```suggestion
        CVarDef.Create("vote.autovote_enabled", false, CVar.SERVERONLY | CVar.ARCHIVE);
```

Можно добавить CVar.ARCHIVE для автоматического сохранения в конфиг

Reply:
если будем менять через команду `> cvar vote.autovote_enabled true`